hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/cluster/SparkExecuteClusteringCommitActionExecutor.java

```diff
@@ -129,7 +131,7 @@ public SparkExecuteClusteringCommitActionExecutor(HoodieEngineContext context,
   /**
    * Validate actions taken by clustering. In the first implementation, we validate at least one new file is written.
    * But we can extend this to add more validation. E.g. number of records read = number of records written etc.
-   * 
+   *
```

Comment:
I have no idea how these are getting included. I reverted this locally and did git add of this file. I pushed it and repeated it twice, but it gets auto formatted somehow :(